motioneye/locale/en/LC_MESSAGES/motioneye.po

```diff
@@ -367,7 +368,7 @@ msgstr "Extra Options"
 
 #: motioneye/templates/main.html:363
 msgid "vi povas aldoni iujn ajn pliajn opciojn ĉi tie por la «motion» demono (uzu la «nomo valoro» formaton, unu opcion por linio)"
-msgstr "You can add any additional options here for the \"motion\" daemon (use the \"name value\" format, one option per line)"
+msgstr "You can add any additional options here for the «motion» daemon (use the «name value» format, one option per line)"
```

Comment:
Hmm, I actually do not agree with this quotation chance done here across all translations. Not sure whether the signs added here are more common in other languages, but certainly not in the ones I know and use. Any other opinion? I'd otherwise try to revert this chance on weblate.

Reply:
Never used those myself, and no English teacher ever has told me to use them.

According to Wikipedia "Guillemets are not conventionally used in the English language." They are listed as being used (more or less) in plenty of languages: https://en.wikipedia.org/wiki/Guillemet#As_quotation_marks, even in Finnish (which is my native language) and for that one I know that it is very rare these days, more like an antiquated style convention. But still, different languages have differing conventions for punctuation, so generally having differing quotation styles in translations would perhaps not be incorrect - as long as the style really is correct (and current) for the language.

Reply:
But I guess here the translator might have (incorrectly) copied the quotation style from Esperanto (where according to Wikipedia it is correct).

Reply:
This also reminds me that I still have not done the Finnish translation :disappointed:

Reply:
This change was done manually by some user, AFAICS. There is no automatism to adjust other translation when one has been changed.

Makes sense to use them in those languages where they are still (more) common than double quotes, or single quotes in case. However, if I find a way to revert this all together for all languages, I'll do that, and people can redo for those individually, where it really makes sense.

Reply:
Oh, I didn't look more at the changes in this PR, but yeah, this "piet" user probably thought it was a good idea and made the effort to change all quotation (unless Weblate somehow makes this kind of thing easy), and now similar effort is needed to revert that :disappointed:

Reply:
Feel free to change them as you like/or roll them back, but this keeps all other translations changes from being upstreamed and also results in the problem described in my other commend: https://github.com/motioneye-project/motioneye/pull/3017#issuecomment-2414762891